tests/typ/visualize/image.typ

```diff
@@ -66,7 +66,7 @@ A #box(image("/files/tiger.jpg", height: 1cm, width: 80%)) B
 #image.decode(`<svg xmlns="http://www.w3.org/2000/svg" height="140" width="500"><ellipse cx="200" cy="80" rx="100" ry="50" style="fill:yellow;stroke:purple;stroke-width:2" /></svg>`.text, format: "svg")
 
 ---
-// Error: 2-168 failed to parse SVG (missing root node)
+// Error: 14-168 failed to parse SVG (missing root node)
```

Comment:
@Dherse 
Why has this changed? :thinking:

Reply:
Because we (@laurmaedje and I) added a lot of new spans all over the place which changed the locations of some errors.